js-miniapp-sdk/src/miniapp.ts

```diff
@@ -66,19 +66,19 @@ export class MiniApp implements MiniAppFeatures, Ad {
     return this.requestPermission(MiniAppPermissionType.LOCATION);
   }
 
-  loadInterstitialAd<T>(id: string): Promise<T> {
-    return (window as any).MiniAppBridge.loadInterstitialAd();
+  loadInterstitialAd(id: string): Promise<null | Error> {
+    return (window as any).MiniAppBridge.loadInterstitialAd(id);
   }
 
-  loadRewardedAd<T>(id: string): Promise<T> {
-    return (window as any).MiniAppBridge.loadRewardedAd();
+  loadRewardedAd(id: string): Promise<null | Error> {
+    return (window as any).MiniAppBridge.loadRewardedAd(id);
   }
 
   showInterstitialAd(id: string): Promise<InterstitialAdResponse> {
-    return (window as any).MiniAppBridge.showInterstitialAd();
+    return (window as any).MiniAppBridge.showInterstitialAd(id);
   }
 
   showRewardedAd(id: string): Promise<RewardedAdResponse> {
-    return (window as any).MiniAppBridge.showRewardedAd();
+    return (window as any).MiniAppBridge.showRewardedAd(id);
```

Comment:
I didn't see the `showRewardedAd` in js bridge. Is it going to be updated later?

Reply:
Yeswe will create on the next PR for update bridge code!